cfme/test_requirements.py

```diff
@@ -472,3 +472,9 @@ def test_quota_alert():
     description='Support of VMWare in MIQ/CFME',
     assignee_id='kkulkarn'
 )
+
+webmks = pytest.mark.requirement(
+    "VM Consoles: VMWare WebMKS",
+    description='VMWare WebMKS support',
+    assignee_id='apagac',
```

Comment:
Aren't you now the assignee?

Reply:
hmmm need to check with Sudhir once about that.

Reply:
allowing this for now, trivial to update.